### PR TITLE
Ie11 arrow fix

### PR DESCRIPTION
**Pull request by brown-a2**
Open/close left-hand arrows did not appear on IE11. This fixes that bug, allowing for arrows to appear on all our supported browsers. Makes use of Gov details component arrows as these don't come with accordion.

### CODE REVIEW

File: src/04-accordion/style.scss [hunk 1]
```diff
@@ -10,9 +10,12 @@
 // https://design-system.service.gov.uk/components/accordion/
 @import "node_modules/govuk-frontend/govuk/components/accordion/accordion";
 
+// https://design-system.service.gov.uk/components/details/
+@import "node_modules/govuk-frontend/govuk/components/details/details";
```
Comment (wilson1000-MoJ) [inline src/04-accordion/style.scss:14]: Hey Adam. Did mean for this import to be here?

File: src/04-accordion/style.scss [hunk 2]
```diff
@@ -62,114 +71,58 @@ SASS modifications to conform GovUK frontend to MoJ block styling
...
 
-    // Add a bottom border to accordion sections (on last one)
+    // Add a bottom border to last accordion section
     &:last-child {
         border-bottom: 1px solid $mojblocks-color-light-grey;
     }
 }
-
-// Replace Gov plus symbol with arrow symbol (when open)
-.js-enabled {
```
Comment (wilson1000-MoJ) [inline src/04-accordion/style.scss:101]: Do we not need js-enabled any more?

**Review by wilson1000-MoJ — APPROVED** [review]
Looks good dude, I've asked a couple of q's

File: src/04-accordion/style.scss (hunk 1 above)
Comment (wilson1000-MoJ) [reply src/04-accordion/style.scss:14]: Actually just read your comment :o)

File: src/04-accordion/style.scss (hunk 2 above)
Comment (brown-a2) [reply src/04-accordion/style.scss:101]: noope, nothing depends on that class now.

File: src/04-accordion/style.scss (hunk 1 above)
Comment (brown-a2) [reply src/04-accordion/style.scss:14]: yes, it's used to bring in the arrows which is part of Toni's designs but unfortunately not part of the gds accordion design.

**Pull request merged**